e2e/support/helpers/e2e-dashboard-helpers.js

```diff
@@ -189,10 +189,6 @@ export function resizeDashboardCard({ card, x, y }) {
   });
 }
 
-export function createLinkCard() {
-  cy.icon("link").click();
-}
```

Comment:
Inlined this little helper because:
- its name was inconsistent with `createTextCard` which does something else
- I needed a `createLinkCard` helper that does similar thing to `createTextCard`

Reply:
I agree with the first point, I run into that myself when I was adding a helper to create a heading card card via the api (see [here](https://github.com/metabase/metabase/pull/34834#discussion_r1365610221)
Should we discuss this in a RFC? I feel like it's good to have have helpers for both creating cards via API and via the UI (sometimes it's more than a `get().click` as they involve dropdowns), but we should find a proper naming for them